### ISSUE TITLE
It works!

### ISSUE THREAD

**Issue by gerroon** [hchapman/reverb#6]
Thanks, I wish the dev did not stop  developing this. It seems like a good concept.

**Comment by MurzNN** [comment]
Confirm, that this app is very good solution! Also there are some forks of this project, that maybe continue developing: https://github.com/hchapman/reverb/network but commits are not so fresh ;(

**Comment by ghost** [comment]
Yes, this works! Here's a quick summary of what I did to get it working. This may help others, as I had to do some searching to get this working correctly. 

1) Download the reverb.apk from the Releases page here on GitHub, install it on your Android phone, and run it. In the app, set the default server to the IP address of your pulseaudio server (probably your desktop PC) using the settings toggle in the upper right 
2) On your Pulseaudio server, open `/etc/pulse/default.pa` in a text editor and add the following: 
`load-module module-native-protocol-tcp auth-anonymous=1 auth-ip-acl=127.0.0.1;<Put your Android phone's IP address here>`
2) Restart pulseaudio via `service` or `systemctl`
3) _Optional:_ To confirm module is loaded: execute `pacmd` on command line and then enter `list-modules` at the pulseaudio command prompt OR install/run command `paman` for a graphical interface
4) Start Reverb on your phone again, it should connect to the default server you set previously.
5) You should see volume controls in Reverb now! Check syslog for errors from pulseaudio if you do not

This worked perfectly for me. Technically, it is not secure. You will see warnings in syslog about this. I am not overly concerned about this on my local network, but a more secure solution would be ideal, eventually. This would require that the Android application sends an authorization cookie file to the Pulseaudio server, which it currently does not do.

**Comment by bernd-wechner** [comment]
That works! My experience is:

1) auth-ip-acl=127.0.0.1 is unnecessary. I don't use it.
2) To save you wondering and looking for it, the v0.5 APK is here: https://github.com/hchapman/reverb/releases
3) The app seems to work if enter the IP of my audio server, but not it's name (i.e. it seems not to do a DNS lookup).

That said it works fine. Brilliant! I do wish we could get a modern build though that can be maintained and improved!
